ts/src/config.ts

```diff
@@ -78,6 +78,12 @@ export interface Config extends GoogleAuthOptions {
   // stack depth may increase overhead of profiling.
   heapMaxStackDepth?: number;
 
+  // Samples with stacks containing this substring will not be included in
```

Comment:
"stacks containing this substring" is unclear. What exact attribute is matched?

Reply:
Tried to clarify this.